build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -231,12 +249,12 @@ fun Test.configureRerun() {
 }
 
 fun Test.determineMaxRetries() = when {
-    project.flakyTestQuarantine.isPresent -> 4
-    else -> 1
+    project.flakyTestStrategy == FlakyTestStrategy.QUARANTINE -> 4
+    else -> 2
```

Comment:
Why increase the number of retries here? Shouldn't that stay `1`?

Reply:
Benedikt said they're using a much higher value (5 maybe? I don't remember correctly). I think it has no harm to increase this value to 2.

Reply:
```suggestion
    else -> 1
```